File: conditionals/Time_of_day.c
//johann hines, template
#include <stdio.h>
#include <time.h>


int main(void){
 time_t now = time(NULL);
struct tm *tm_struct = localtime(&now);
int hour = tm_struct->tm_hour;
printf ("the time is %d\n", hour);
if (hour <= 12  ){
printf("Goodmorning user");
    }else if (hour < 12  && hour > 7  ){
printf ("good afternoon user");
    }else{ 
printf ("goodnight user");
    return 0;
}
}

File: conditionals/time of day.py
#Johann Hines, time of day python.
import time 

current =time.time()
local_time = time.localtime(current)
minutes = local_time.tm_min
Hour = local_time.tm_hour
if Hour <= 12:
    print("Goodmorning user")
elif Hour < 12 & Hour < 6 : 
    print("Good afternnon user")
else:
    print("Goodnight user")